filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -91,27 +91,52 @@ The following example configures Filebeat to ignore all the files that have a `g
   exclude_files: [".gz$"]
 -------------------------------------------------------------------------------------
 
+===== tags
+
+A list of tags that the Beat includes in the `tags` field of each published
+event. Tags make it easy to select specific events in Kibana or apply
+conditional filtering in Logstash. These tags will be appended to the list of
+tags specified in the `shipper` configuration.
+
+Example:
+
+[source,yaml]
+--------------------------------------------------------------------------------
+filebeat:
+  prospectors:
+    - paths: ["/var/log/app/*.json"]
+      tags: ["json"]
+--------------------------------------------------------------------------------
+
 [[configuration-fields]]
 ===== fields
 
-Optional fields that you can specify to add additional information to the output. For
-example, you might add fields that you can use for filtering log data. Fields can be
-scalar values, arrays, dictionaries, or any nested combination of these. All scalar values will be interpreted as strings. By default,
-the fields that you specify here will be grouped under a `fields` sub-dictionary in the output document. To store the custom fields as top-level fields, set the `fields_under_root` option to true.
+Optional fields that you can specify to add additional information to the
+output. For example, you might add fields that you can use for filtering log
+data. Fields can be scalar values, arrays, dictionaries, or any nested
+combination of these. All scalar values will be interpreted as strings. By
```

Comment:
I removed this sentence from the docs. I didn't see your comment prior to asking you the question on Slack.